code/mob/living/silicon/robot.dm

```diff
@@ -962,22 +962,17 @@
 			if (src.opened && user) boutput(user, "You must close the cover to swipe an ID card.")
 			else if (src.wiresexposed && user) boutput(user, "<span class='alert'>You need to get the wires out of the way.</span>")
 			else
-				sleep (6)
-				if (prob(50))
-					if (user)
-						boutput(user, "You emag [src]'s interface.")
-					src.visible_message("<font color=red><b>[src]</b> buzzes oddly!</font>")
-					src.emagged = 1
-					src.handle_robot_antagonist_status("emagged", 0, user)
-					if(src.syndicate)
-						src.antagonist_overlay_refresh(1, 1)
-					SPAWN_DBG(0)
-						update_appearance()
-					return 1
-				else
-					if (user)
-						boutput(user, "You fail to [ locked ? "unlock" : "lock"] [src]'s interface.")
-					return 0
+				if (user)
+					boutput(user, "You emag [src]'s interface.")
+				src.visible_message("<font color=red><b>[src]</b> buzzes oddly!</font>")
+				src.emagged = 1
+				src.handle_robot_antagonist_status("emagged", 0, user)
+				if(src.syndicate)
+					src.antagonist_overlay_refresh(1, 1)
+				SPAWN_DBG(0)
+					update_appearance()
```

Comment:
is this really needed to be inside a spawn

Reply:
Having looked at update_appearance, I doubt it?